### ISSUE TITLE
Ability to set output colors

### ISSUE THREAD

**Issue by orestesgaolin** [joonro/Get-ChildItem-Color#18]
Hi! Thanks for such a great extension to Powershell - it is really helping in daily work.

I was wondering if it would be possible to change the output colors on the go? Perhaps similarly to the Oh-My-Posh solution where we have a set of variables available to customize.

```bash
> $ThemeSettings.Colors

Name                           Value
----                           -----
VirtualEnvForegroundColor      White
VirtualEnvBackgroundColor      Red
PromptHighlightColor           DarkBlue
PromptForegroundColor          White
GitForegroundColor             Black
SessionInfoBackgroundColor     Black
GitDefaultColor                DarkGreen
PromptSymbolColor              White
```

**Comment by joonro** [comment]
Hi - I'm glad that it is helpful! Thanks for kind words. 

That sounds like a great idea and should be not too hard. I can take a look at what Oh-My-Posh does and try to incorporate a similar settings when I get a chance. Thanks!

**Comment by joonro** [comment]
This is implemented by #23. Will update README soon with a new release.
